test/consumer.test.ts

```diff
@@ -0,0 +1,247 @@
+import { config, expect } from 'chai';
+import { SecurityEngine, DiagnosticsPipeline, codeActionsMap } from '../src/consumers';
+
+describe('Response consumer test', () => {
+    
+    it('Consume response for free-users', async () => {
```

Comment:
Non the test uses await, so no need for async

Reply:
Ack!